packages/loader/driver-utils/src/documentStorageServiceProxy.ts

```diff
@@ -32,16 +32,12 @@ export class DocumentStorageServiceProxy implements IDocumentStorageService {
         return this.internalStorageService.getVersions(versionId, count);
     }
 
-    public async read(blobId: string): Promise<string> {
-        return this.internalStorageService.read(blobId);
-    }
-
     /**
-     * {@inheritDoc @fluidframework/driver-definitions#IDocumentStorageService.readString}
+     *
+     * @deprecated - only here for back compat, will be removed after release
      */
-    public async readString(blobId: string): Promise<string> {
-        const base64Result = await this.read(blobId);
-        return fromBase64ToUtf8(base64Result);
+    public async read(blobId: string): Promise<string> {
+        return bufferToBase64(await this.readBlob(blobId));
```

Comment:
Similar to feedback in other place, please use 
        return this.internalStorageService.read(blobId);